dev/ms/create-secret-manager-secret.yaml

```diff
@@ -14,3 +14,8 @@ secrets:
       - kubeSecretKey: stairwaypassword
         path: secret/dsde/datarepo/dev/helm-datarepodb-dev
         vaultKey: stairwaypassword
+  - secretName: azure-ms
```

Comment:
Could we remove Mark's `ms` directory?

Reply:
Yeah, we should probably take some time to remove everything at some point.  I'm hesitant in case there's any reference deployment stuff right away but I'll create a ticket

Reply:
Thanks!